app/routerTransition.js

```diff
@@ -45,7 +45,7 @@ const filterAndSortURLs = (count, latencies) => {
     return urls;
 };
 
-const willTransitionTo = (nextState, replaceState, callback) => {
+const willTransitionTo = (nextState, replaceState, callback, appInit=true) => { //appInit is true when node is manually selected in access settings
```

Comment:
Shouldn't this say that appInit is false when selecting nodes in the settings?

Reply:
No because appInit should only happen when it's not be user choice. I'm using this to differentiate between user picking a server and server being chosen when the app is loaded.

Reply:
Exactly, so when a user picks a server appInit is false, but on first load of the app it is true, which is the opposite of what the comment says. You are calling it with false on line 36

Reply:
Lolz. Yea, just need to fix the comment.